app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
` Ruby on Rails Tutorial Sample App`
Tìm hiểu và sử dụng I18n  nhé

=> dùng `I18n cho TOÀN BỘ TEXT ở tất cả các file html` nhé